src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_IParameterReferenceExpression.cs

```diff
@@ -756,18 +760,15 @@ public void M(int x)
 }
 ";
             string expectedOperationTree = @"
-ISwitchStatement (1 cases) (OperationKind.SwitchStatement) (Syntax: 'switch (x) ... }')
```

Comment:
👍 I think there is a bug tracking the issue that you fixed here. We can close all such bugs later..